tutorials/hist/hsumTimer.C

```diff
@@ -12,14 +13,14 @@
 ///
 /// \author Rene Brun
 
-Float_t ratio;
+Float_t ratioratio;
```

Comment:
I believe you get a compilation error, not an exception, right?
What about `progressRatio`?